### PR TITLE
Fix OpenJDK7 failures on Travis

### PR DESCRIPTION
**Pull request by krmahadevan**
More details can be found in this issue:

https://github.com/gradle/gradle/issues/2421

Fixes # .

### Did you remember to?

- [ ] Add test case(s)
- [ ] Update `CHANGES.txt`

We encourage pull requests that:

* Add new features to TestNG (or)
* Fix bugs in TestNG

If your pull request involves fixing SonarQube issues then we would suggest that you please discuss this with the 
[TestNG-dev](https://groups.google.com/forum/#!forum/testng-dev) before you spend time working on it.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 sudo: false
 language: java
+dist: precise
```
Comment (juherr) [inline .travis.yml]: I didn't try but maybe it is possible to force `precise` for java7 only: 
```
     - jdk: oraclejdk7
       dist: precise
     - jdk: openjdk7
       dist: precise
```

Comment (krmahadevan) [reply .travis.yml]: Tried that. It still failed after 8 mins. Again no reason why it fails on JDK8. Please see [here](https://travis-ci.org/cbeust/testng/jobs/271425668)

Comment (krmahadevan) [reply .travis.yml]: @juherr - I think that did the trick. The JDK8 build failed due to test failures. Guess there was some momentary glitch in retrieving the logs. Can you please help get this merged ?

**Comment by juherr** [comment]
@cbeust I merged myself because it fixes the build which is a good help for reviews.

**Pull request merged**